### PR TITLE
fix: wrong public keys signing 

### PR DESCRIPTION
**Pull request by QuantumExplorer**
## Issue being fixed or feature implemented
There was an issue with signing public keys that has now been resolved. Previously keys were signing only the key which is not secure. Now they are signing the whole state transition.

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated relevant unit/integration/functional/e2e tests
- [X] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by strophy** [comment]
What was the issue?

File: packages/rs-platform-serialization/src/lib.rs [hunk 1]
```diff
@@ -290,35 +290,144 @@ pub fn derive_platform_signable(input: TokenStream) -> TokenStream {
...
+                                None
+                            }
+                        })
+                        .find_map(|stream| {
+                            let mut iter = stream.into_iter();
+                            if let Some(proc_macro2::TokenTree::Ident(ident)) = iter.next() {
+                                if ident == "into" {
+                                    if let Some(proc_macro2::TokenTree::Literal(lit)) = iter.next()
+                                    {
+                                        println!("found into: {:?}", lit);
```
Comment (markin-io) [inline packages/rs-platform-serialization/src/lib.rs:337]: Should we remove this?

Comment (QuantumExplorer) [reply packages/rs-platform-serialization/src/lib.rs]: Yep

File: packages/rs-dpp/src/identity/state_transition/identity_create_transition/identity_create_transition.rs [hunk 2]
```diff
@@ -65,15 +66,16 @@ pub struct SerializationOptions {
 pub struct IdentityCreateTransition {
     #[serde(rename = "type")]
     pub transition_type: StateTransitionType,
-    // Own ST fields
-    pub public_keys: Vec<IdentityPublicKeyInCreationWithWitness>,
+    // The signable
+    #[platform_signable(into = "Vec<IdentityPublicKeyInCreationSignable>")]
```
Comment (shumkov) [inline packages/rs-dpp/src/identity/state_transition/identity_create_transition/identity_create_transition.rs:70]: proc macros are cool!

**Review by markin-io — APPROVED** [review]

**Review by shumkov — CHANGES_REQUESTED** [review]

**Pull request merged**